cpp/include/cudf/strings/find.hpp

```diff
@@ -140,6 +140,36 @@ std::unique_ptr<column> contains(
   rmm::cuda_stream_view stream      = cudf::get_default_stream(),
   rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Returns a table of columns of boolean values for each string where true indicates
+ * the target string was found within that string in the provided column.
+ *
+ * Each column in the result table corresponds to the result for the target string at the same
+ * ordinal. i.e. 0th column is the boolean-column result for the 0th target string, 1th for 1th,
+ * etc.
+ *
+ * If the target is not found for a string, false is returned for that entry in the output column.
+ * If the target is an empty string, true is returned for all non-null entries in the output column.
+ *
+ * Any null string entries return corresponding null entries in the output columns.
+ * e.g.:
+ *   input:   "a", "b", "c"
+ *   targets: "a", "c"
+ *   output is a table with two boolean columns:
+ *     column_0: true, false, false
+ *     column_1: false, false, true
```

Comment:
This will format poorly. Use the `@code/@endcode` doxygen tags.

Reply:
done.